package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@signoz-icons/icons",
+  "version": "1.0.0",
+  "description": "SVG-based React icon components",
+  "keywords": [
+    "React",
+    "SVG",
+    "Icons"
+  ],
+  "module": "dist/index.esm.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "prebuild": "rimraf src && rimraf dist",
```

Comment:
will `rm -rf src && rm -rf dist` not work here?

Reply:
@ahmadshaheer This will work but rimraf is often preferred because it's a cross-platform solution.